tools/src/test/java/org/apache/kafka/tools/streams/ResetStreamsGroupOffsetTest.java

```diff
@@ -738,4 +755,26 @@ private static void produceMessagesOnTwoPartitions(final int numOfMessages, fina
             data
         );
     }
+
+    /**
+     * Executes the StreamsGroupCommand with the given arguments and validates the exit code.
+     * <p>
+     * This helper method is used to test scenarios where the command is expected to exit
+     * with a specific status code (e.g., 0 for success, 1 for an error). It captures the
+     * exit code by using a mock {@link Exit.Procedure} and asserts that it matches the
+     * expected value.
+     *
+     * @param args             The command-line arguments to pass to the StreamsGroupCommand.
+     * @param expectedExitCode The expected exit code from the command execution.
+     */
+    private static void validateResetOffsetsExitCode(String[] args, int expectedExitCode) {
```

Comment:
It looks like there is some duplciate code in this package. Could you try to eliminate it?

Reply:
Let's get rid of it entirely? -> https://github.com/apache/kafka/pull/20293#discussion_r2308814874